### DISCUSSION TITLE
Non-GSA Backups

### DISCUSSION THREAD

**Discussion by scarecr0w12** [gameserverapp/Platform#547] [Suggestions]
I would love the options to setup my own backups, either via FTP, Rsync, SCP, or whatever method, so that way i can schedule more frequent backups, and have them stored where they are directly accessible to me. Or have the options for local backups, as i for example have 2 Drives in my box, so i could use my secondary as a backup storage. Many other panel systems allow for setting up of these things and would love it to come to GSA

Having purely "Cloud" based backups, without direct access, and that only backup at non-configurable times/periods, is against most Disaster recovery methods, and does not provide a REAL rock solid backup methodology

**Comment by mvaessen** [comment]
Hi, you are free to setup your own backup methods. These can be setup outside of GSA. There are existing suggestions posted on here for that. I recommend checking those out.

Closing this suggestion as it is out-of-scope.
